aries_vcx_core/src/anoncreds/credx_anoncreds.rs

```diff
@@ -28,14 +30,34 @@ use credx::{
     ursa::cl::MasterSecret as UrsaMasterSecret,
 };
 use indy_credx as credx;
+use serde::{de::DeserializeOwned, Deserialize, Serialize};
 use serde_json::Value;
 use uuid::Uuid;
 
 use super::base_anoncreds::BaseAnonCreds;
 
-const CATEGORY_CREDENTIAL: &str = "VCX_CREDENTIAL";
 const CATEGORY_LINK_SECRET: &str = "VCX_LINK_SECRET";
 
+const CATEGORY_CREDENTIAL: &str = "VCX_CREDENTIAL";
+const CATEGORY_CRED_DEF: &str = "VCX_CRED_DEF";
+const CATEGORY_CRED_KEY_PROOF: &str = "VCX_CRED_DEF_PROOF";
+const CATEGORY_CRED_DEF_PRIV: &str = "VCX_CRED_DEF_PRIV";
+const CATEGORY_CRED_SCHEMA: &str = "VCX_CRED_SCHEMA";
+const CATEGORY_CRED_SCHEMA_ID: &str = "VCX_CRED_SCHEMA_ID";
```

Comment:
IMO this record type is pretty confusing. It's not intuitive that this is used to map cred_def_ids to schema_ids. I'm hopeful that we can remove  `CATEGORY_CRED_SCHEMA_ID` soon, as it seems unnecessary.

But if we don't want to remove it right now, perhaps you could add a comment to make it more obvious what this record type's purpose is?

Reply:
I'll modify the name.